round4/team-18.md

```diff
@@ -0,0 +1,106 @@
+# Chrome Extension Idea: Chromagotchi
+
+## Authors
+
+Clara Fee, John Otto, Laya Yalamanchili, Via Liu 
+
+## Problem Statement

```

Comment:
Great proposal. I particularly liked how you used your user stories to highlight the various features that your extension offers (eg.  Tamagotchi profiles, categorizing websites,  the disable feature, the goal setting, break implementation… ) and how exactly that they would provide value to the various types of users and stake holders (eg. the user, their parents, the different types of workers).

One selling point that was a bit unclear was how your extension would encourage users to close their unused tabs. I would recommend adding a user story that explains this.